optimal_guide_finder/cas_model.py

```diff
@@ -230,5 +241,11 @@ def _identify_target_sequences_matching_pam(self, pam_seq, positions_at_mers, fu
                 # Does not account for circular DNAs
                 if begin > 0 and end < len(full_sequence):
                     target_sequence = full_sequence[begin: end]
-                    target_sequence_list.append((target_sequence, nt))
-        return target_sequence_list
+                    if set(target_sequence).issubset('ATCG'):
```

Comment:
I thought 'ATCG' would need to be a set to be able to compare [t.issubset(set('ATCG')], if it works as is though should be good.